Development_Documents/Client_Release_Process.md

```diff
@@ -0,0 +1,96 @@
+- Coordinate the builds of the client
+- Coordinate the testing of each build of the client
+- As bugs are found during testing, identify which ones will be fixed and included in the current release vs deferred to a future release
+- Ensure that all required items that need to be updated in the release have been updated (see list of items produced in a release in the overview above)
+- Call for a consensus vote of committers to determine if a build is ready to be declared as generally available
+Throughout the process, the release manager is expected to communicate what is going on as the work progresses and make sure to address concerns that are raised as the work progresses.
+
+### Build Master
+Someone needs to be responsible for building the clients for each of Windows, Mac, Linux, and Android.  It can be a different person or the same person for each platform. These people will be responsible for creating official builds when the release manager requests a new build.  Build masters will work with the PMC in order to gain access to any signing keys or login rights required to create builds for their platforms.  
+The build masters will work with the release manager in order to ensure that new builds are created at the requested times as the release work proceeds.
+
+### Testers
+People who want to help test the client releases should be familiar with the [Alpha Testing](https://boinc.berkeley.edu/trac/wiki/AlphaInstructions) wiki page and associated processes.
+
+## Process
+### Prepare for a Release
+The release manager will announce the intention to make a new release of the client.  When they do this they should make a request to the community to find out if there is any work in progress that will be available shortly that should be included on the request.  The release manager will work with anyone who responds to determine if that work will be included or not depending on the relevant concerns at the time.
+
+The release manager should also connect with the build masters and confirm with them that they are available for creating builds during the next several weeks as the work progresses.
+
+### Release Branch
+The release branch is created once the release manager has decided that all work that will be included in the release is available in the master branch.
+
+The release branch is considered feature frozen.  Only bug fixes designated by the release manager will be merged into the release branch.
+
+Bug fixes designated by the release manager for fixing within the release should be branched from the release branch and merged directly into the release branch.  Doing this reduces the chance of inadvertently bringing in unrelated code changes.
+
+If the release manager determines that some code should be merged from master into the release branch, then this can be done.  However, this should be viewed as an unusual step and carefully considered due to the risk of introducing unexpected change.
```

Comment:
We need to be careful with the terms used to avoid confusion: the notion of "some code" doesn't make (a lot of) sense in the context of a "merge". A merge **always** comprises **all** code leading up to the commit you merge.